rlbottraining/common_exercises/kickoff_exercise.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass, field
+from typing import Union
+from pathlib import Path
+from math import pi
+
+from rlbot.utils.game_state_util import GameState, BallState, CarState, Physics, Vector3, Rotator
+from rlbot.matchconfig.match_config import Team, PlayerConfig
+
+from rlbottraining.training_exercise import TrainingExercise, Playlist
+from rlbottraining.grading.grader import Grader
+from rlbottraining.rng import SeededRandomNumberGenerator
+from rlbottraining.common_graders.kickoff_grader import QuickResetKickoffGrader
+
+
+@dataclass
+class KickoffType:
+    pos: Vector3
+    rot: Rotator
+
+
+# Default kickoffs.
+right_corner = KickoffType(Vector3(-2048, -2560, 18), Rotator(0, 0.25*pi, 0))
+left_corner = KickoffType(Vector3(2048, -2560, 18), Rotator(0, 0.75*pi, 0))
+back_right = KickoffType(Vector3(-256, -3840, 18), Rotator(0, 0.5*pi, 0))
+back_left = KickoffType(Vector3(256.0, -3840, 18), Rotator(0, 0.5*pi, 0))
+straight = KickoffType(Vector3(0, -4608, 18), Rotator(0, 0.5*pi, 0))
+
+
+@dataclass
+class KickoffExercise(TrainingExercise):
+    grader: Grader = field(default_factory=QuickResetKickoffGrader)
+    kickoffs: Union[tuple, list] = ()

```

Comment:
I'd write it as `spawns: List[SpawnLocations] = field(default_factory=list)` since I don't see the value of tuples here.
note: `from typing import List`